unity/Assets/StreamingAssets/content/MoM/hj/monsters.ini

```diff
@@ -0,0 +1,39 @@
+[MonsterWarlock]
+name={ffg:MONSTER_WARLOCK}
+image="{import}/img/Monster_Warlock"
+traits=humanoid
+awareness=4
+horror=3
+brawn=2
+health=4
+healthperhero=1
```

Comment:
I an not sure about the monster health and how healthperhero realy works. Does health per hero means for 1 more hero the health is raised by 1 or does it mean that if healthperhero=2, then every two heroes the health is raised by one.
The app really shows different results. I thought I have seen a DimensionalShambler with a health of 6 using two investigators, which raised to 8 when using three.

Reply:
Formless Spawn does spawn in What Lies Within, but this is a special, non regular monster.

Reply:
I think that you have health and healthperhero correct.  Dimensional Shambler starts with health of 6 (two investigators is the minimum allowed) and adds two per hero so health with three heroes = 8.